Assigment3/test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+user_one:
+  email: MyString1@example.com
+  username: MyStringp
+  password_digest: $2a$12$oM56UPgGS2eqXsjZALxR0ebd1EUrazQpZFxIzfwMhG8Gbg5R5LfDW
```

Comment:
I think it would be much easier to set the `password`, rather than the `password_digest`. You can just do
```
user_one:
  email: MyString1@example.com
  username: MyStringp
  password: password123

user_two:
  email: MyString@example.com
  username: MyString
  password: password123
```

So if you want to test the log in for these users, you know what password to send as a parameter